### ISSUE TITLE
Can the authors provide the hyper-parameters in training step?

### ISSUE THREAD

**Issue by onlyonewater** [ketatam/DiffDock-PP#20]
Hi, @ketatam , thanks for releasing the code, I have a question: Now I want to reproduce the results on the DIPS test set, so I directly run the command `bash ./src/train.sh`, but I find that the hyper-parameters in `dips_esm.yaml` are different from the original paper, in the `dips_plus.yaml`, the epoch is 2000, but in your original paper and README.md in this repo, you said you set epoch as 170, and the  `args.yaml` in the pre-trained model you provide, the ns, nv are 32, 6, but in the `dips_esm.yaml`, the ns, nv are 16, 4. so it makes me very confused which parameters should I use. Can the authors tell me which parameters were used when they did the experiment?

**Comment by onlyonewater** [comment]
and the code may have some bugs when using multi-GPU training, I think someone who uses Multi-GPU to train the model must note: 1. the data may not move to GPU, because the author set if torch.cuda.is_available() is True and num_gpus==1, only in this situation, the data may move to GPU, and 2. in the dataloader, when num_gpus > 1, it uses DataListLoader, so if you directly use batch.cuda(), it may cause an error, because the list type does not have .cuda attribute

**Comment by onlyonewater** [comment]
I think the authors should release a reliable code. anyone can reproduce your results with your provided code.

**Comment by onlyonewater** [comment]
for anyone who wants to reproduce the results from scratch:
1. if you have GPU memory less than 24GB, and you also want to train a new docking model, you should filter some samples that have more than **1000** residues. But if you want to reproduce the performance that the paper mentioned, 48GB of GPU memory is all you need.
2.  if you want to achieve the performance that the paper mentioned, you need to change the ns, nv parameters in dips_esm.yaml. The detailed parameters you should see in the ./checkpoints/large_model_dips/args.yaml
3. set the no_graph_cache as False in the config file, it can save you time in the data process (save the debug time), and you can also set the sample_train as False in the config file, which can save you training time.
